filebeat/filebeat.reference.yml

```diff
@@ -161,6 +161,16 @@ filebeat.modules:
     #var.paths:
 
 
+#------------------------------- kibana Module -------------------------------
+- module: {{ module }}
```

Comment:
Hmmm... This seems to be a problem of the generator. Do you mind changing it manually now. I am opening a PR with the fix in the meantime.